Update SubscriptionManagement.xml

Rnhatch opened the pull request:
Fixed action sequences and removed some standard table functions that did not work (Edit Row,  Add Row) on permission set tab table.

Rnhatch committed:
Update SubscriptionManagement.xml

Fixed action sequences and removed some standard table functions that did not work (Edit Row,  Add Row) on permission set tab table.

josephine-the-great reviewed (changes requested):
When a license is revoked, the counts at the top of the page don't update. You may need to requery the PackageLicense model after that "revoke license" action or update formulas or something.

Other than that looks good. Thanks for doing this!

Rnhatch committed:
Update SubscriptionManagement.xml

Added Requery for high level package information model so summary block updates after license is revoked. 

Added ModelLookup formula field on user model to identify users in list that Do or Don't have Skuid Licenses - and conditionally rendered the "Deactivate License" button

Rnhatch committed:
Update SubscriptionManagement.xml

Update Name field in user table so link goes to User Detail in setup.

josephine-the-great reviewed (approved):
Looks good to go!
